docs/dockstore-logo-usage-guidelines.rst

```diff
@@ -0,0 +1,129 @@
+  :width: 256
+
+.. |Horizontal Black| image:: /assets/images/Dockstore_Logos_and_Icons/Dockstore-logo-horizontal-black.png
+  :width: 256
+
+.. |Vertical White Blue| image:: /assets/images/Dockstore_Logos_and_Icons/Dockstore-logo-vertical-white-blue.png
+  :width: 128
+
+.. |Vertical Black Blue| image:: /assets/images/Dockstore_Logos_and_Icons/Dockstore-logo-vertical-black-blue.png
+  :width: 128
+
+.. |Horizontal White Blue| image:: /assets/images/Dockstore_Logos_and_Icons/Dockstore-logo-horizontal-white-blue.png
+  :width: 256
+
+.. |Horizontal Black Blue| image:: /assets/images/Dockstore_Logos_and_Icons/Dockstore-logo-horizontal-black-blue.png
+  :width: 256
+
+.. |Icon Blue| image:: /assets/images/Dockstore_Logos_and_Icons/Dockstore-icon-blue.png
+  :width: 64
+
+.. |Icon Black| image:: /assets/images/Dockstore_Logos_and_Icons/Dockstore-icon-black.png
+  :width: 64
+
+.. |Icon White| image:: /assets/images/Dockstore_Logos_and_Icons/Dockstore-icon-white.png
+  :width: 64
+
+===============================
+Dockstore Logo Usage Guidelines
+===============================
+
+About
+=====
+It is important to establish a sense of branding throughout our publicly facing, digital media. Currently, we have several themes / layouts, navigation systems, font sets, color palettes. We would like to unify our designs under a single set of standards.
+
+Color
+=====
+The official Dockstore color is:
+
+  .. list-table:: Dockstore Blue
+
+    * - |Dockstore Dot|
+      - Hex: #7986CB
+      - R: 121, G: 134, B: 203
+
+When using the Dockstore logo or icon, the acceptable colors are Dockstore Blue, black, or white as shown in the “Logo” section of this document.
+
+If the surrounding color is visually unappealing with Dockstore Blue, please use the black or white variants as appropriate.
+
+Logos
+=====
+
+  .. tabularcolumns:: |c|c|c|
+
+  .. list-table::
+    :header-rows: 1
+
+    * - Horizontal
+      - Vertical
+      - Icon
+    * - |Horizontal blue|
+      - |Vertical blue|
+      - |Icon blue|
+    * - Dockstore-logo-horizontal-blue
+      - Dockstore-logo-vertical-blue
+      - Dockstore-icon-blue
+    * - |Horizontal black|
+      - |Vertical black|
+      - |Icon black|
+    * - Dockstore-logo-horizontal-black
+      - Dockstore-logo-vertical-black
+      - Dockstore-icon-black
+    * - |Horizontal white|
+      - |Vertical white|
+      - |Icon white|
+    * - Dockstore-logo-horizontal-white
+      - Dockstore-logo-vertical-white
+      - Dockstore-icon-white
+    * - |Horizontal black blue|
+      - |Vertical black blue|
+      -
+    * - Dockstore-logo-horizontal-black-blue
+      - Dockstore-logo-vertical-black-blue
+      -
+    * - |Horizontal white blue|
+      - |Vertical white blue|
+      -
+    * - Dockstore-logo-horizontal-white-blue
+      - Dockstore-logo-vertical-white-blue
+      -
+
+A zip file containing these logos in both PNG and SVG formats can be found :download:`here </assets/images/Dockstore_Logos_and_Icons.zip>`.
```

Comment:
If we're going to make high-res logos, they probably belong here, and a note should mention it includes high-res.

Reply:
If we are going high-res we should probably also do small, at least for the icon.  Small tends to be more problematic than large due to the loss of visual elements when downscaling.